test_main.py

```diff
@@ -3,97 +3,44 @@
 # To run the tests, use: python3 -m pytest --capture=sys
 
 from database import Database
+from filters import Filter
+import filters
 from main import main
 from test_utils import region_dict
+import random
 
 db = Database()
 print(len(db))
 
 
-def test_no_args(capsys):
-    main([__file__])
-    out, err = capsys.readouterr()
-    assert out.startswith("No command line arguments specified.")
-
-
-def test_three_args(capsys):
-    main([__file__, 1, 2, 3])
-    out, err = capsys.readouterr()
-    assert out.startswith("Invalid number of arguments.")
-
-
-def test_two_letters(capsys):
-    main([__file__, 'bu'])
-    out, err = capsys.readouterr()
-    assert 'Butterfree' in out
-    # prefix search only
-    main([__file__, 'ut'])
-    out, err = capsys.readouterr()
-    assert 'butterfree' not in out.lower()
-
-
 def test_extra(capsys):
-    main([__file__, 'extra'])
-    out, err = capsys.readouterr()
-    assert out.count('Castform') == 3, out  # issue #89
-    assert 'turtwig' not in out.lower()
+    main(['-e'])
+    # TODO: Assertion based on number of files on ./Extras
+    assert str(random.choice(Filter.filtered_list)).startswith('---')
 
 
 def test_region_names(capsys):
-    main([__file__, 'regions'])
-    out, err = capsys.readouterr()
-    for region_name in region_dict:
-        assert region_name in out
-
-
-def test_help(capsys):
```

Comment:
I mean bro, if we can't trust python's stdlib we need to start testing sums, boolean operations and built-in functions. Some assumptions must be made in order to test stuff.

I mean if we need to test that, we just might as well create test_python and see if basic arithmetic works